seqr/utils/elasticsearch/constants.py

```diff
@@ -99,6 +99,9 @@
     'gnomad_genomes': {
         'filter_AF': ['gnomad_genomes_FAF_AF', 'gnomad_genomes_AF_POPMAX_OR_GLOBAL'],
     },
+    'gnomad_genome_svs': {
+        'filter_AF': ['gnomad_svs_AF'],
```

Comment:
`filter_AF` only is relevant if we want to use a different field for filtering than for displaying AF, which is not the case here. You want to change this config to just `'gnomad_svs': {}`